app/views/layouts/application.html.erb

```diff
@@ -7,7 +7,13 @@
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
-
+    <% facade = HeaderFacade.new %>
```

Comment:
should we be creating the HeaderFacade object here in the view? Part of me thinks this functionality should be put in a controller that then returns a `@facade` object for the view to use.